packages/components/src/components/Flyout/Flyout.stories.tsx

```diff
@@ -0,0 +1,193 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+
+import { Flyout, FlyoutContent, FlyoutRoot } from './Flyout';
+
+type Story = StoryObj<typeof meta>;
+const meta: Meta<typeof FlyoutContent> = {
+    component: FlyoutContent,
+    subcomponents: {
+        // @ts-expect-error - Storybook has the wrong type for TooltipContent
+        'Flyout.Root': FlyoutRoot,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        children: 'Hello World',
+    },
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <Flyout.Header showCloseButton>Header</Flyout.Header>
+                    <Flyout.Body {...args} />
+                    <Flyout.Footer>
+                        <div className="tw-flex tw-justify-end tw-gap-2">
+                            <Button emphasis="default">Cancel</Button>
+                            <Button>Submit</Button>
+                        </div>
+                    </Flyout.Footer>
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+export default meta;
+
+export const WithHeaderAndFooter: Story = {
+    name: 'With Header and Footer',
+};
+export const BasicContent: Story = {
+    name: 'Basic Content',
+    args: {
+        children: 'Hello World',
+    },
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <Flyout.Body {...args} />
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+
+export const WithFooter: Story = {
+    name: 'Button Content',
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <Flyout.Body>I am a flyout</Flyout.Body>
+                    <Flyout.Footer>
+                        <Button emphasis="default">Cancel</Button>
+                        <Button>Submit</Button>
+                    </Flyout.Footer>
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+
+export const WithHeader: Story = {
```

Comment:
Let's move it before `WithFooter` so it is hierarchically correct

Reply:
changed